src/Boo.Lang.Compiler/Boo.Lang.Compiler.csproj

```diff
@@ -25,6 +26,7 @@
     <DefineConstants>TRACE;DEBUG;IGNOREKEYFILE,NET_2_0</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
+    <Prefer32Bit>false</Prefer32Bit>

```

Comment:
Good questions :) : Since i wrote it several months ago i don't remember exactly, probably I needed to use the "collectible" boo in a project of mine in .NET 4.5 and therefore i used VS with .NET 4.5 without thinking too much, this explains the .NET 4.5 thing and [this blog post](http://blogs.microsoft.co.il/sasha/2012/04/04/what-anycpu-really-means-as-of-net-45-and-visual-studio-11/) explains the Prefer32bit one (it's the new AnyCPU default when using VS 11 on .NET 4.5).

Probably it's better to use 4.5 and remove those Prefer32Bit from csprojs, i'll try to take care of them and report here if something else prevents me to do it.